src/pages/LetterEdit/index.ts

```diff
@@ -148,10 +145,9 @@ class LetterEdit extends Component {
       console.error('Saved letter cannot be found!');
       notyf.error(_('An error occured while trying to retrieve updated state'));
     } else {
-      this.state.letter = updatedLetterState;
+      this.state.letter.lastUpdate = updatedLetterState.lastUpdate;  // INFO: do NOT update translatedElements (related to T0319)
```

Comment:
Remove unnecessary comment
```suggestion
      this.state.letter.lastUpdate = updatedLetterState.lastUpdate;
```